app/js/services/situationService.js

```diff
@@ -120,6 +120,7 @@ angular.module('ddsCommon').factory('SituationService', function($http, $session
             }
 
             var result = {
+                referenceId: situation._id,
```

Comment:
I'm not sure OpenFisca's “reference” means what you think it means :wink:
See https://github.com/openfisca/openfisca-core/pull/525